composer.json

```diff
@@ -91,10 +91,12 @@
     },
     "scripts": {
         "post-install-cmd": [
-            "DrupalComposer\\DrupalScaffold\\Plugin::scaffold"
+            "DrupalComposer\\DrupalScaffold\\Plugin::scaffold",
+            "patch -d docroot <patches/htaccess.patch"
         ],
         "post-update-cmd": [
-            "DrupalComposer\\DrupalScaffold\\Plugin::scaffold"
+            "DrupalComposer\\DrupalScaffold\\Plugin::scaffold",
+            "patch -d docroot <patches/htaccess.patch"
```

Comment:
does there need to be a ` ` in here before the patch path? `docroot < patches/htaccess.patch`

Reply:
Updated those, all good